server/game/cards/01-Core/KingsHuntingParty.js

```diff
@@ -3,25 +3,18 @@ const DrawCard = require('../../drawcard.js');
 class KingsHuntingParty extends DrawCard {
     setupCardAbilities(ability) {
         this.persistentEffect({
-            condition: this.effectCondition.bind(this),
+            condition: () => this.anyOpponentHasKing(),
```

Comment:
It's a little weird to wrap a function call in a lamda when you could just pass the function itself, but on my list of things I care about, this is pretty low

Reply:
The rules for `this` are so screwed up that I'm not 100% that's true in this case. But either way the lambda reads better to me than `this.method.bind(this)`.

Reply:
binding this means you get to use this in the function, if the function doesnt need this then it's not needed.  The need for binding this is going away in a future node version i believe